config/schemas/config.yaml

```
@@ -4772,10 +4734,31 @@ properties:
           > [!note]
           > See [the upstream documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) for reference.
         type: array
+      useralertmanager:
+        title: user Alertmanager
+        description: Configuration options for deploying a user-specific Alertmanager
```

As this will already be a breaking change then we can update this to follow [the glossary](https://elastisys.io/welkin/glossary/#application-developer).
```suggestion
      devAlertmanager:
        title: Application Developer Alertmanager
        description: Configuration options for deploying an application developer-specific Alertmanager
```
Also for config keys use camelCase.

Also add to the description that config common with the service cluster Alertmanager can be configured via `.alertmanagerSpec`.

---

Done with this commit https://github.com/elastisys/compliantkubernetes-apps/commit/184e21b9de730b7aa42587078114679512b53ae0